apps/api/.env.cloud-run.example

```
@@ -0,0 +1,28 @@
+
+# ── Per-environment plain env vars ──
+EMAIL_ADMIN_DESTINATIONS=admin@f3nation.com
+EMAIL_FROM="F3 Support <support@f3nation.com>"
+EMAIL_REGION_IN_A_BOX_CC=
+GOOGLE_LOGO_BUCKET_BUCKET_NAME=f3-logos
+GOOGLE_LOGO_BUCKET_PROJECT_ID=f3data
+NEXT_PUBLIC_API_URL=https://staging.api.f3nation.com
+NEXT_PUBLIC_AUTH_URL=https://staging.auth.f3nation.com
+NEXT_PUBLIC_CHANNEL=staging
+NEXT_PUBLIC_GA_MEASUREMENT_ID=
+NEXT_PUBLIC_MAP_URL=https://staging.map.f3nation.com
```

dotenv-linter (4.0.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)